.github/pull_request_template.md

```diff
@@ -14,5 +14,5 @@ Please ensure your pull request adheres to the following guidelines:
 Fixes: #issue-number
 
 ```release-note
-<!-- Enter the release note text here if needed or remove this section! -->
```

Comment:
I think we should simplify the overall release note template. First-time contributors often keep the entire text in there and we end up removing it again.

Reply:
Makes sense, any more specific ideas? I'm not yet familiar with what we have there to suggest anything... Maybe we could just get rid of `release-note/misc` and make it an implicit default?

Personally, I do think that release note review is a good idea anyway, sometimes you want to adjust the order, or even tweak the style, especially for major and minor releases.

Reply:
>Maybe we could just get rid of release-note/misc and make it an implicit default?

having a default, without requiring the developer to add one, will make some, if not all, PRs that should have a proper release note to end up with the `release-note/misc`.